templates/hoover.nomad

```diff
@@ -166,102 +169,67 @@ job "hoover" {
       }
     }
   }
-
-  group "collections" {
-    task "nginx" {
+  group "traefik-internal" {
+    task "traefik-internal" {
       driver = "docker"
+      config {
+        image = "traefik:1.7"
+        port_map {
+          http = 80
+          api = 8080
```

Comment:
Let's call this `admin` or something more descriptive, and use the same name everywhere